charts/yurt-app-manager/templates/admission-webhooks/job-patch/role.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.admissionWebhooks.enabled .Values.admissionWebhooks.patch.enabled (not .Values.admissionWebhooks.certManager.enabled) }}
```

Comment:
why not combine `role.yaml` into `clusterrole.yaml`?

Reply:
Use different file, it's easy maintian, and other projects use the same way.

Reply:
ok